rafthttp/http.go

```diff
@@ -135,16 +135,14 @@ func (h *streamHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	case path.Join(RaftStreamPrefix, string(streamTypeMessage)):
 		t = streamTypeMessage
 	default:
-		log.Printf("rafthttp: ignored unexpected streaming request path %s", r.URL.Path)
-		http.Error(w, "invalid path", http.StatusNotFound)
+		http.Error(w, "invalid path", http.StatusBadRequest)
```

Comment:
the prefix part of URL path. For example, 'v2/key' part of '/v2/key/0001' and 'v2' part of '/v2/0001'. I don't know the good naming for it.